types/react-native-vector-icons/index.d.ts

```diff
@@ -1,6 +1,7 @@
-// Type definitions for react-native-vector-icons 4.2
+// Type definitions for react-native-vector-icons 4.3
 // Project: https://github.com/oblador/react-native-vector-icons
 // Definitions by: Kyle Roach <https://github.com/iRoachie>
+//                 Tim Wang <https://github.com/timwangdev>
```

Comment:
Adding your name here means you sign up as a maintainer, and you'll be notified of all issues, PRs for the library. Just making sure you know